app/jenkins-for-jira-ui/src/components/JenkinsServerList/JenkinsServerList.tsx

```diff
@@ -56,29 +65,44 @@ const JenkinsServerList = (): JSX.Element => {
 		</ButtonGroup>
 	);
 
-	return jenkinsServers?.length ? (
-		<>
-			<div className={headerContainer}>
-				<PageHeader actions={pageHeaderActions}>Jenkins configuration</PageHeader>
-			</div>
+	let contentToRender;
 
-			<StyledDescription>
-				After you connect your Jenkins server to Jira and send a deployment
-				event from your CI/CD tool, you will be able to view development
-				information within your linked Jira issue and view deployment pipelines
-				over a timeline with insights.
-			</StyledDescription>
+	if (jenkinsServers?.length && moduleKey === 'jenkins-for-jira-ui-admin-page') {
+		contentToRender = (
+			<>
+				<div className={headerContainer}>
+					<PageHeader actions={pageHeaderActions}>Jenkins configuration</PageHeader>
+				</div>
+				<StyledDescription>
+					After you connect your Jenkins server to Jira and send a deployment
+					event from your CI/CD tool, you will be able to view development
+					information within your linked Jira issue and view deployment pipelines
+					over a timeline with insights.
+				</StyledDescription>
+				<ConnectedServers jenkinsServerList={jenkinsServers} refreshServers={fetchAllJenkinsServers} />
+			</>
+		);
+	} else if (moduleKey === 'get-started-page') {
+		contentToRender = (
+			<>
+				<div className={headerContainer}>
+					<PageHeader>Jenkins configuration</PageHeader>
+				</div>
+				<Spinner size='large' />
```

Comment:
Before the redirect kicks in, the empty/configured state was rendering briefly which is definitely not what we want. Accounted for that by rendering this spinner (matches what we see while waiting for the page to load).